components/automate-ui/src/app/helpers/auth/authorized.spec.ts

```diff
@@ -232,21 +232,21 @@ describe('AuthorizedComponent real round trip', () => {
           userperms: () => ({
             status: Status.loadingSuccess,
             byId: <IndexedEntities<UserPermEntity>>{
+              '/apis/iam/v2/projects/project-1/rules': {
+                id: '/apis/iam/v2/projects/project-1/rules',
+                get: false, put: false, post: true, delete: false, patch: false
+              },
+               '/apis/iam/v2/projects/project-2/rules': {
+                id: '/apis/iam/v2/projects/project-2/rules',
+                get: false, put: false, post: false, delete: false, patch: false
+              },
               '/apis/iam/v2/users': {
                 id: '/apis/iam/v2/users',
-                get: false,
-                put: false,
-                post: false,
-                delete: false,
-                patch: false
+                get: false, put: false, post: false, delete: false, patch: false
```

Comment:
Curious about the inlining-- did lint grab that? I found the other way a bit easier to read

Reply:
eh, having fewer opinions since spec file :ignoreme:

Reply:
Not a lint thing; just a preference: did not feel that a few pieces of data should take a whole screenful. 🤷